android/app/src/main/java/com/reactnativenavigation/views/RnnToolBar.java

```diff
@@ -56,6 +56,7 @@
     private DrawerLayout mDrawerLayout;
     private ActionBarDrawerToggle mDrawerToggle;
     private ArrayList<View> mMenuItems;
+    private int mButtonColor;
```

Comment:
@guyca When calling `setNavUpButton` without a screen it's implied we're navigating back.  Having no screen object also uses the default color when styling the icon, which may not have been the color of the previous screen's menu icon.  This may just be more of an argument for a previous comment of mine mentioning splitting up `setNavUpButton` into separate style and state methods.